cloud_info/providers/openstack.py

```diff
@@ -281,10 +286,14 @@ def get_templates(self):
                 flavor_id = str(getattr(flavor, flavor_id_attr))
                 template_id = '%s%s#%s' % (URI, tpl_sch,
                                            OpenStackProvider.occify(flavor_id))
+                flavor_ib_v = flavor.get_keys().get(flavor_ib_k, False)
+                template_ib = (True if flavor_ib_v == extra_specs_ib['value']
```

Comment:
This line can be changed to `template-ib = flavor_ib_v == extra_specs_ib['value']` (the rest is not needed)